tidb-cloud/data-service-manage-data-app.md

```diff
@@ -13,21 +13,41 @@ This document describes how to manage your Data Apps in the TiDB Cloud console.
 
 To create a Data App for your project, perform the following steps:
 
-1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
-2. In the left pane, click <MDSvgIcon name="icon-create-data-app" /> **Create DataApp** and update the default name if necessary.
-3. The newly created Data App is added to the top of the list. A default `untitled endpoint` is created for the new Data App.
+1. On the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project, click <MDSvgIcon name="icon-create-data-app" /> **Create DataApp** in the left pane.
+2. Enter a name for the Data App, and select clusters that you want the Data App to access.
+3. (Optional) To automatically deploy metadata and endpoints of the Data App to your preferred GitHub repository and branch, expand **Connect your Data App to GitHub**, and then do the following:
+
+    1. Click **Install on GitHub**, and then follow the on-screen instructions to install **TiDB Cloud Data Services** as an application on your target repository.
+    2. Click **Authorize** to authorize access to the application on GitHub.
+    3. Specify the target repository, branch, and directory where you want to save the configuration files of your Data App.
+
+    > **Note:**
+    >
+    > - The directory must start with a slash. For example, `/mydata`. If the directory you specified does not exist in the target repository and branch, it will be created automatically.
+    > - The combination of repository, branch, and directory represents the path of the configuration files, which must be unique among Data Apps. If the path already contains files of existing Data Apps, you need to specify a new path instead.
+    > - If you have created a GitHub branch based on configuration files of an existing Data App and want to import the configuration of the existing Data App to a new Data App, see [Import configurations of an existing Data App](#import-configurations-of-an-existing-data-app).
+
+4.Click **Create Data App**.
+
+The newly created Data App is added to the top of the list. A default `untitled endpoint` is created for the new Data App.
+
+If you have connected your Data App to GitHub, check your selected directory on GitHub. If the configuration files of the Data App have been committed by `tidb-cloud-data-services`, it indicates that your Data App is connected to GitHub successfully. For more information, see [Deploy Automatically with GitHub](/tidb-cloud/data-service-manage-github-integration.md).
```

Comment:
`tidb-cloud-data-service`

Reply:
updated in https://github.com/pingcap/docs/pull/13674/commits/6f89b99e025de9d35f1f5de7be3d4ec6212519c3